docs/elements.md

```diff
@@ -706,22 +706,19 @@ generation if two (or more) choices are identical.
-| `allow-blank`                 | boolean | false   | Whether or not an empty submission is allowed. If `allow-blank` is set to `true`, a submission that does not select any option will be marked as incorrect instead of invalid. |
+| Attribute                    | Type                                      | Default  | Description                                                                                                                                                                    |
+| ---------------------------- | ----------------------------------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `answers-name`               | string                                    | —        | Variable name to store data in. Note that this attribute has to be unique within a question, i.e., no value for this attribute should be repeated within a question.           |
+| `weight`                     | integer                                   | 1        | Weight to use when computing a weighted average score over elements.                                                                                                           |
+| `display`                    | "block", "inline", or "dropdown"          | "block"  | Display option for the input field. Block and inline display answer choices as radio buttons, while dropdown presents option as a dropdown.                                    |
+| `number-answers`             | integer                                   | special  | The total number of answer choices to display. Defaults to displaying one correct answer and all incorrect answers.                                                            |
+| `order`                      | "random", "ascend", "descend", or "fixed" | "random" | Order to display answer choices. Fixed order displays choices in the same order as the original source file.                                                                   |
+| `hide-letter-keys`           | boolean                                   | false    | Hide the letter keys in the answer list, i.e., (a), (b), (c), etc.                                                                                                             |
+| `all-of-the-above`           | string                                    | `false`  | Add "All of the above" choice. See below for details.                                                                                                                          |
+| `none-of-the-above`          | string                                    | `false`  | Add "None of the above" choice. See below for details.                                                                                                                         |
```

Comment:
Nit: other string options are documented with "..." instead of `...`.

Reply:
Changed this to match other enum-like choices for this same element: https://github.com/PrairieLearn/PrairieLearn/pull/8903/commits/40b4ab19ae602f71800741c38fa0f0ac603d60bf